Sources/BraveWallet/Crypto/Transaction Confirmations/TransactionConfirmationView.swift

```diff
@@ -63,16 +63,26 @@ struct TransactionConfirmationView: View {
     )
     .overlay(
       Group {
-        TransactionStatusView(
-          confirmationStore: confirmationStore,
-          networkStore: networkStore,
-          transactionDetails: $transactionDetails) {
-            if confirmationStore.unapprovedTxs.count == 0 {
-              onDismiss()
-            }
-            // update activeTransactionId
-            confirmationStore.updateActiveTxIdAfterSignedClosed()
+        if confirmationStore.isTxSubmitting {
+          ZStack {
+            Rectangle()
+              .fill(Color(.braveGroupedBackground))
+              .edgesIgnoringSafeArea(.all)
+            ProgressView()
+              .progressViewStyle(.braveCircular(size: .large, tint: .braveBlurpleTint))
```

Comment:
Just use `ProgressView` + `frame` + `background` instead of `ZStack`

```swift
ProgressView()
  .progressViewStyle(.braveCircular(size: .large, tint: .braveBlurpleTint))
  .frame(maxWidth: .infinity, maxHeight: .infinity)
  .background(Color(.braveGroupedBackground).ignoresSafeArea())
```

Reply:
Also just FYI `edgesIgnoringSafeArea` is deprecated and we should switch to using `ignoresSafeArea` in new code

Reply:
updated in [8d59b95](https://github.com/brave/brave-ios/pull/6960/commits/8d59b95439fdfcd0da5f733c63d51fea80b2a899)